flow.py

```diff
@@ -15,3 +21,15 @@ def __str__(self):
             str(self.data_amt) + ", Start time: " + str(self.start_time) + ">"
 
     __repr__ = __str__
+
+    def startFlow(self):
+        # Figure out how many packets to send.
+        # This is a fixed amount P = data_amt/pkt_size
+        # Where pkt_size is 1024 bytes.
+
+        # =====TODO: put PACKET_SIZE as a global variable for reference?====
+
+        num_packets = (int)(ceil(self.data_amt * 1.0e6 / 1024))
+        for i in range(num_packets):
+            pkt = packet.Packet(self.source, self.destination, i)
+            enqueue(event.SendPacket(1, pkt, self.source.link))
```

Comment:
Unsure about sending at same time...should be OK before congestion control is implemented.